adapters/appnexus.go

```diff
@@ -235,3 +237,42 @@ func NewAppNexusAdapter(config *HTTPAdapterConfig, externalURL string) *AppNexus
 		usersyncInfo: info,
 	}
 }
+
+func CollectImpData(oreq openrtb.BidRequest, req *pbs.PBSRequest, unit pbs.PBSAdUnit) string {
```

Comment:
This can be private (start with a lower-case letter).

As a general rule, it's a good idea to minimize the visibility of everything (functions, structs, variables, etc). If the language guarantees that some block of code is only used in limited scope, refactoring it is much easier later.